src/data/fintype/order.lean

```diff
@@ -171,3 +171,10 @@ begin
   obtain ⟨b, rfl⟩ := ha,
   exact hM b,
 end
+
+/-! ### Induction -/
```

Comment:
Not sure you need a section for a single result that might not even be needed in the future.